solutions/cf-920E.mdx

```diff
@@ -0,0 +1,233 @@
+---
+id: cf-920E
+source: CF
+title: Connected Components?
+author: Daniel Qiu
+---
+
+
+[Official Editorial](https://codeforces.com/blog/entry/57516)
+
+
+## Explanation
+## Solution 1 - Faster DFS
+
+Doing a standard DFS would be too slow. To speed it up, we have to do two
+optimizations.
+
+### Optimization 1
+
+You can store whether there is no edge between two nodes using an array of hashsets, instead of a adjacency list. If the input tells you that nodes $a$ and $b$ do node have an edge between them, you would insert node $a$ at $\texttt{array}[b]$, and node $b$ and $\texttt{array}[a]$ 
```

Comment:
Since your array name for the array of hashets is `adj` in the implementation, also make the name `adj` here for consistency

```suggestion
You can store whether there is no edge between two nodes using an array of hashsets, instead of a adjacency list. If the input tells you that nodes $a$ and $b$ do not have an edge between them, you would add $a$ to $\texttt{adj}[b]$, and $b$ to $\texttt{adj}[a]$ 
```